Dear/Dear/Model/Data.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Data.swift
+//  Dear
+//
+//  Created by 한연희 on 2022/07/18.
+//
+import Foundation
+
+struct LetterItem: Codable {
+    //병원 정보
+    let hospistal: String
+    //편지 내용
+    let title: String
+    let description: String
+    //작성 완료여부 체크
```

Comment:
```suggestion
    /// 작성 완료여부 체크
```
`///`을 하면 자동완성 때 뜬답니다 : )

Reply:
앗 감사합니다